### ISSUE TITLE
Color contrast of Label of disabled dropdown fails test

### ISSUE THREAD

**Issue by taparikh** [OfficeDev/office-ui-fabric-react#11385]
### Environment Information

- **Package version(s)**: 7.64.2
- **Browser and OS versions**: Version 78.0.3904.108 (Official Build) (64-bit)

### Describe the issue:
Color contrast test fails on label color for disabled dropdowns. 

### Please provide a reproduction of the issue in a codepen:
The issue present on the official office ui fabric website

https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown

#### Actual behavior:
Color contrast test fails on label color of disabled dropdown
![image](https://user-images.githubusercontent.com/51679982/70275936-0407a600-1764-11ea-96b6-be6da3b37286.png)

Here are the details of this bug
![image](https://user-images.githubusercontent.com/51679982/70275978-18e43980-1764-11ea-934b-d834cf18a979.png)

#### Expected behavior:
The color contrast should not be this low. 

### Documentation describing expected behavior
![image](https://user-images.githubusercontent.com/51679982/70275991-1bdf2a00-1764-11ea-9e79-c258ca4c3244.png)

Title: WCAG 1.4.3: Elements must have sufficient color contrast (#Dropdown448-label)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Elements must have sufficient color contrast (color-contrast - https://dequeuniversity.com/rules/axe/3.3/color-contrast?application=msftAI)

Target application: Home - Office UI Fabric - https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown

Element path: #Dropdown448-label

Snippet: <label class="ms-Label ms-Dropdown-label label-311" id="Dropdown448-label">Disabled example with defaultSelectedKey</label>

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 2.63 (foreground color: #a19f9d, background color: #ffffff, font size: 10.5pt (14px), font weight: bold). Expected contrast ratio of 4.5:1

Environment: Chrome version 78.0.3904.108

====

This accessibility issue was found using Accessibility Insights for Web 2.10.3 (axe-core 3.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by xugao** [comment]
thanks @taparikh for reporting, will look into it

**Comment by xugao** [comment]
@taparikh , please see responses in this issue #10902

> according to accessibility guidelines:
> "Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement."
> 

So I will conclude that this is not an issue according to the guidelines above.

**Comment by taparikh** [comment]
This is not a decoration text. This fails the fast pass accessibility test and hence needs to be fixed/

**Comment by taparikh** [comment]
Can we reopen this issue ?

**Comment by micahgodbolt** [comment]
it is not decoration text, but it is part of an inactive user interface component. @betrue-final-final i know that content inside of a disabled input does not need to meet those guidelines, but is that also true for the label?

Is there a way that we need to mark up a disabled label? <label disabled>? Or <label for="disabledControlID"> ?

**Comment by xugao** [comment]
@taparikh  sorry i missed your message!
@micahgodbolt  thanks for following up here :)

**Comment by xugao** [comment]
@betrue-final-final is on vacation. @ecraig12345 what's your opinion on this?

**Comment by xugao** [comment]
@ecraig12345 helped found an old issue 
https://github.com/OfficeDev/office-ui-fabric-react/issues/9909
https://github.com/microsoft/accessibility-insights-web/issues/981

this confirms that this is not a bug on the fabric side but rather it's a false negative from the a11y tests.

@taparikh pls help close this issue if this answers you

**Comment by taparikh** [comment]
@xugao  This issue is different from the one you have specified. The color of the label of a disabled dropdown is failing the color contrast check

**Comment by ecraig12345** [comment]
Despite being a different component it's the same root cause.

**Comment by taparikh** [comment]
However, the fast pass of the accessiblity insights tool marks this as failed. Which means that the label (not placeholder) of a disabled component should meet color contrast guidelines?

**Comment by xugao** [comment]
@taparikh - no, there is no accessibility issue here. this is a test issue. pls read through this bug: https://github.com/microsoft/accessibility-insights-web/issues/981

**Comment by micahgodbolt** [comment]
Thanks for catching that @xugao! Glad we got a resolution here. j

**Comment by micahgodbolt** [comment]
making as "not an issue". External usually means the bug is 'caused' by an external tool. In this case there is no bug, it's just a false positive created by an external tool.

**Comment by taparikh** [comment]
Closing this bug as it is a false positive
